docs/resources/integration_aws_eventbridge.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: "cloudamqp"
+page_title: "CloudAMQP: cloudamqp_integration_aws_eventbridge"
+description: |-
+  Creates and manages an AWS EventBridge for a CloudAMQP instance.
+---
+
+# cloudamqp_integration_aws_eventbridge
+
+This resource allows you to create and manage, an [AWS EventBridge](https://aws.amazon.com/eventbridge/) for a CloudAMQP instance. Once created continue and mapping the EventBridge in the [AWS Eventbridge console](https://console.aws.amazon.com/events/home).
+
+~>  If messages are too large (maximum body size allowed is 256kb) or are not valid JSON, AWS EventBridge will not accept it and the message will be rejected (Tips: setup a dead-letter queue to catch them).
```

Comment:
Our Console says

> Note that our consumer needs to have exclusive usage to the configured queue and the maximum body size allowed on msgs by AWS is 256kb. The message body has to be valid JSON for AWS Eventbridge to accept it. If messages are too large or are not valid JSON, they will be rejected (tip: setup a dead-letter queue to catch them).

Should we include the wording about "consumer needs to have exclusive usage" here too?

Reply:
Yes, we could use the whole section.

Reply:
Updated the documentation with the same text as our Console.